lib/mountain_view/presenter.rb

```diff
@@ -10,10 +10,10 @@ def initialize(slug, properties = {})
       @properties = default_properties.deep_merge(properties)
     end
 
-    def render(context)
+    def render(context, &block)
       context.extend ViewContext
       context.inject_component_context self
-      context.render partial: partial
+      context.render partial, partial: partial, block: yield
```

Comment:
Curious about passing the partial name twice, `render partial, partial: partial`, what was the reason for that?